### ISSUE TITLE
Logitech G920 detected, but nothing works

### ISSUE THREAD

**Issue by Leniwcowaty** [berarma/oversteer#147]
The wheel is being detected and is showing in "Device" section, but apart of that, nothing works. Axis are not being displayed (this is with one pedal pressed and wheel maxed out to left):
![image](https://github.com/berarma/oversteer/assets/32055243/0bb816e1-0893-40e4-926e-54cdc46d5125)

FFB not working at all, cannot load profile, getting error:
![image](https://github.com/berarma/oversteer/assets/32055243/bbc9456f-efbb-449a-abcc-0fae267e372f)

Tests also not working, nothing happens.
Fedora 38, kernel 6.4.7, all dependencies installed, checked every USB port in my PC. It just doesn't work :/

**Comment by berarma** [comment]
Which version is this? Latest master?

Can you please try the latest release?

**Comment by Leniwcowaty** [comment]
Yes, this is the latest release, installed an hour before the post

**Comment by berarma** [comment]
Just before releasing 0.7.2 I closed #103 and everything should be working according to that ticket. Can you confirm you're using that version?

Also running `oversteer --debug` from the terminal could give me more information.

**Comment by Leniwcowaty** [comment]
![image](https://github.com/berarma/oversteer/assets/32055243/31413c2c-bbdd-48dd-81ce-8db33cbe477e)


As you can see - 0.7.2. As for debug, after starting:

`DEBUG:root:update_device_list: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0007/input/input21/event19 /dev/input/event19
DEBUG:root:/dev/input/event19: ID_VENDOR_ID: 046d ID_MODEL_ID: c262
DEBUG:root:Devices: {'/devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0007/input/input21/event19': <oversteer.device.Device object at 0x7f958d1f2450>}
DEBUG:matplotlib:matplotlib data path: /usr/share/matplotlib/mpl-data
DEBUG:matplotlib:CONFIGDIR=/home/kacper/.config/matplotlib
DEBUG:matplotlib:interactive is False
DEBUG:matplotlib:platform is linux
DEBUG:matplotlib:CACHEDIR=/home/kacper/.cache/matplotlib
DEBUG:matplotlib.font_manager:Using fontManager instance from /home/kacper/.cache/matplotlib/fontlist-v330.json`


When clicking "Start new" in tests:

`Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/oversteer/gtk_handlers.py", line 259, in on_test_start_clicked
    self.controller.start_test()
  File "/usr/local/lib/python3.11/site-packages/oversteer/gui.py", line 475, in start_test
    self.test = Test(self.device, test_callback)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/oversteer/test.py", line 10, in __init__
    self.input_device = self.device.get_input_device()
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get_input_device'`


After toggling "Center wheel" (autocentering works):

`DEBUG:root:Setting autocenter strength: 65535
DEBUG:root:Setting autocenter strength: 0`
Then a lot of Setting autocenter strength when I modify the slider.


When I unplug wheel:
`003:046D:C262.0007/input/input21/event19
DEBUG:root:remove: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0007/input/input21/js0
DEBUG:root:remove: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0007/input/input21`


When I plug again:

`DEBUG:root:add: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0008/input/input22
DEBUG:root:add: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0008/input/input22/js0
DEBUG:root:add: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0008/input/input22/event19
DEBUG:root:update_device_list: /devices/pci0000:00/0000:00:02.1/0000:05:00.0/0000:06:08.0/0000:0c:00.0/0000:0d:0c.0/0000:13:00.0/usb3/3-3/3-3:1.0/0003:046D:C262.0008/input/input22/event19 /dev/input/event19
DEBUG:root:/dev/input/event19: ID_VENDOR_ID: 046d ID_MODEL_ID: c262`

I hope this is helpful for you, for me it's just a lot of gibbrish.

**Comment by berarma** [comment]
The latest master commit has still version 0.7.2. You need to know whether you used the zip release file or the master branch.

I think you have used the master branch. I've uploaded a fix for this issue. Can you please test it?

**Comment by Leniwcowaty** [comment]
I used the method of installation provided in README - cloned the git repo and used master branch.
I installed it now again and it works as expected - wheel is detected, shows everything. Sadly two more problems arise - the method with launching it with game (`oversteer -p [profile] -g "%command%"`) doesn't work with Flatpak Steam (`/bin/sh: line 1: /usr/local/bin/oversteer: No such file or directory`) and with normal Steam, from repos, it works, launches Oversteer and applies my autocentering options, but no FFB in BeamNG. I suppose the problem is strictly with this game, bcs ETS 2 and ATS have FFB even without oversteer. 
Anyway, thank you for your help and keep up the great work!
